### PR TITLE
Work on the sessions page in the admin area

### PR DESCRIPTION
**Pull request by adrianbanks**
This add some styling to the table of sessions, as well as the ability to sort the table.

### CODE REVIEW

File: DDDEastAnglia/App_Start/BundleConfig.cs [hunk 1]
```diff
@@ -7,53 +7,70 @@ public class BundleConfig
...
-                        "~/Scripts/jquery.unobtrusive*",
-                        "~/Scripts/jquery.validate*"));
+            AddScriptBundle(bundles, "~/bundles/jqueryval",
+                            "~/Scripts/jquery.unobtrusive*",
+                            "~/Scripts/jquery.validate*");
 
-            bundles.Add(new ScriptBundle("~/bundles/bootstrap").Include(
-                        "~/Scripts/bootstrap.js"));
+            AddScriptBundle(bundles, "~/bundles/tablesorter", 
+                            "~/Scripts/jquery.tablesorter.js");
```
Comment (alastairs) [inline DDDEastAnglia/App_Start/BundleConfig.cs]: Is this the minified file? If so, you might as well add a copy of the file called `query.tablesorter.min.js` as well, so the Web Optimisation framework doesn't need to do any processing.

File: DDDEastAnglia/Content/admin.css [hunk 2]
```diff
@@ -0,0 +1,43 @@
+﻿.admin-table {
```
Comment (alastairs) [inline DDDEastAnglia/Content/admin.css]: There's a bunch of styles in Bootstrap for styling tables. See http://twitter.github.io/bootstrap/base-css.html#tables

File: DDDEastAnglia/App_Start/BundleConfig.cs (hunk 1 above)
Comment (alastairs) [inline DDDEastAnglia/App_Start/BundleConfig.cs]: Scrap that, just looked through the rest of the pull request. You just need to update the files-to-include path to read `"~/Scripts/jquery.tablesorter*"`

File: DDDEastAnglia/Content/admin.css (hunk 2 above)
Comment (adrianbanks) [inline DDDEastAnglia/Content/admin.css]: Couldn't remove them all, as some are needed by the table sorter.

**Comment by adrianbanks** [comment]
I've just rebased this, but it still won't auto-merge here. I've just merged it manually into my local and there were no conflicts though.

**Comment by alastairs** [comment]
Weird. I'll try too.

## 

I got some auto-merges (i.e., conflicts that were resolved without manual intervention), which make GitHub think the pull request can't be merged automatically.

**Comment by alastairs** [comment]
The deployment to the staging site of this pull request failed: 

```
error : Copying file sorter\asc.gif failed. Could not find file 'sorter\asc.gif'.
```

Please could you fix up the branch and open a fresh pull request for it?

Full deployment log: 

```
remote: New deployment received.
remote: Updating branch 'staging'.
remote: Updating submodules.
remote: Preparing deployment for commit id '1b818d3557'.
remote: Generating deployment script.
remote: info:    Executing command site deploymentscript
remote: info:    Project file path: .\DDDEastAnglia\DDDEastAnglia.csproj
remote: info:    Solution file path: .\DDDEastAnglia.sln
remote: info:    Generating deployment script for .NET Web Application
remote: info:    Generated deployment script files
remote: info:    site deploymentscript command OK
remote: Running deployment command...
remote: Handling .NET Web Application deployment.
remote:   All packages listed in packages.config are already installed.
remote:   DDDEastAnglia -> C:\DWASFiles\Sites\dddeastaging\VirtualDirectory0\site\repository\DDDEastAnglia\bin\DDDEastAnglia.dll
remote:   Transformed Web.config using C:\DWASFiles\Sites\dddeastaging\VirtualDirectory0\site\repository\DDDEastAnglia\Web.Release.config into obj\Release\TransformWebConfig\transformed\Web.config.
remote:   Copying all files to temporary location below for package/publish:
remote:   C:\DWASFiles\Sites\dddeastaging\Temp\748e4382-e5b4-4261-90c9-6c1f26b3be45.
remote: D:\Program Files (x86)\MSBuild\Microsoft\VisualStudio\v11.0\Web\Microsoft.Web.Publishing.targets(3094,5): error : Copying file sorter\asc.gif to C:\DWASFiles\Sites\dddeastaging\Temp\748e4382-e5b4-4261-90c9-6c1f26b3be45\sorter\asc.gif failed. Could not find file 'sorter\asc.gif'. [C:\DWASFiles\Sites\dddeastaging\VirtualDirectory0\site\repository\DDDEastAnglia\DDDEastAnglia.csproj]
remote: An error has occurred during web site deployment.
remote: 
remote: Error - Changes committed to remote repository but your website not updated.
```

**Comment by adrianbanks** [comment]
https://github.com/dddeastanglia/DDDEastAnglia/pull/95

**Pull request merged**